rules-reviewed/eap8/eap7/javax-to-jakarta-package.windup.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="javax-to-jakarta-package"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides analysis of Java EE applications that need to change their import statements to replace javax.* with jakarta.*
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+        </dependencies>
+        <targetTechnology id="eap" versionRange="[8,)" />  
+        <targetTechnology id="jakarta-ee" versionRange="[9,)" />      
+    </metadata>
+    <rules>
+        <!-- https://issues.jboss.org/browse/WINDUP-3152 -->
+        <rule id="javax-to-jakarta-import-00001">
+            <when>
+                <javaclass references="javax.{renamed}{*}" as="javaClass">
+                    <location>IMPORT</location>
+                </javaclass>
+            </when>
+            <perform>
+                <iteration over="javaClass">
+                    <hint title="javax.{renamed} has been replaced by jakarta.{renamed}" effort="1"
+                          category-id="mandatory">
+                        <message>Replace the `jaxax.{renamed}` import statement with `jakarta.{renamed}` </message>
```

Comment:
Spelling mistake in javax

Reply:
Hi Shveta,
We managed to catch that and fix it in a subsequent PR
If you have a look at the source on Master for https://github.com/windup/windup-rulesets/blob/master/rules-reviewed/eap8/eap7/javax-to-jakarta-package.windup.xml and https://github.com/windup/windup-rulesets/blob/master/rules-reviewed/eap8/eap7/tests/javax-to-jakarta-package.windup.test.xml
You will see the correct javax 
Thanks,
Phil